modules/test/playwright/pages/FDS/FDS.page.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect} from '@playwright/test';
+
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/applicationsMenu.page';
+
+import type {Page} from '@playwright/test';
+
+export class FDSPage {
+	page: Page;
+	applicationsMenuPage: ApplicationsMenuPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+	}
+
+	async goto() {
+		await this.applicationsMenuPage.goToFDS();
```

Comment:
Not sure if I like this or not... :thinking: 

On the one hand, it makes sense to use pages inside pages because that allows the test to decouple from ApplicationsMenuPage and delegate how to navigate to the FDS page to FDSPage.

But on the other hand, if we couple pages too much between them we could end up having problems and additionally, by adding this layer of indirection we are making the test less obvious, maybe (because reading the test it is not obvious that FDSPage.goto is clicking on the applications menu).

Reply:
And then, for this very specific case, if you declare a gotoFDS method in ApplicationsMenuPage, declaring FDSPage.goto seems a bit redundant and it can lead to confusion, maybe.

It would be different if we had created a generic goto method in ApplicationsMenuPage that is invoked with the proper parameters from FDSPage.goto. That would be much better.

I.e, I'm proposing FDSPage.goto to invoke a code similar to:

```
this.applicationMenuPage.goto('Data Sets')
```

Reply:
I am leaning towards composition, the current approach, even at the cost of lesser code readability:
1. It reduces code duplication.
2. I like the separation of concerns where Locators are used only on the page where they is on. In this case, I'm talking about `FDSMenuItem: Locator;`. If we implemented standalone navigation in `FDSPage`, we would need to duplicate and resolve the menu item Locator here.

Reply:
I'm not sure either, but I didn't innovate much here, I just got inspired by [objects](https://github.com/liferay/liferay-portal/blob/fff775e599f3515c078fb8c504e13a15b7fabb99/modules/test/playwright/pages/object/objectDefinitions.page.ts#L24).